python/cuda_parallel/tests/test_reduce.py

```diff
@@ -63,40 +63,15 @@ def op(a, b):
     for num_items in [42, 420000]:
         h_input = np.random.random(num_items) + 1j * np.random.random(num_items)
         d_input = numba.cuda.to_device(h_input)
-        temp_storage_bytes = reduce_into(None, d_input, d_output, None, h_init)
+        temp_storage_bytes = reduce_into(None, d_input, d_output, len(d_input), h_init)
```

Comment:
Same comment. Consider saving and reusing the result of `len(d_input)`. 

BTW, is `d_input` necessarily viewed as 1d array? In NumPy, `len(arr)` gives `arr.shape[0]`, but here we likely need `arr.size`.

Reply:
I've gone ahead and replaced `len(arr)` with `arr.size`, here and in other places, as part of https://github.com/NVIDIA/cccl/pull/3718/commits/b8474a49358f818d1322195b7e3b82ba1b79a25e.